cpp/src/slice2swift/Gen.cpp

```diff
@@ -774,7 +774,7 @@ Gen::TypesVisitor::visitDictionary(const DictionaryPtr& p)
     const string ostr = getUnqualified("Ice.OutputStream", swiftModule);
     const string istr = getUnqualified("Ice.InputStream", swiftModule);
 
-    const string optionalFormat = getUnqualified(getOptionalFormat(p), swiftModule);
```

Comment:
`getOptionalFormat` returns enumerators like `.FSize`, `.F2`, etc.
without qualification, so calling `getUnqualified` was pointless.
Unrelated to the bug, but just something I stumbled across and decided to fix.